charts/workspacemanager/templates/cert.yaml

```diff
@@ -6,11 +6,27 @@ metadata:
   labels:
     {{ template "workspacemanager.labels" . }}
 spec:
-  secretName: {{ .Chart.Name }}-cert-secret
+  secretName: {{ .Chart.Name }}-cert
   renewBefore: {{ .Values.certManager.renewBefore }}
   dnsNames:
   - {{ template "workspacemanager.fqdn" . }}
   issuerRef:
     name: {{ .Values.certManager.issuerName }}
     kind: {{ .Values.certManager.issuerKind }}
+{{ else if .Values.vaultCert.enabled -}}
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
+metadata:
+  name: secretdefinition-{{ .Chart.Name }}-cert
+  labels:
+    {{ template "workspacemanager.labels" . }}
+spec:
+  name: {{ .Chart.Name }}-cert
+  keysMap:
+    tls.crt:
+      path: {{ required "A valid vaultCert.cert.path value is required when vaultCert is enabled" .Values.vaultCert.cert.path }}
+      key: {{ required "A valid vaultCert.cert.secretKey value is required when vaultCert is enabled" .Values.vaultCert.cert.secretKey }}
+    tls.key:
+      path: {{ required "A valid vaultCert.key.path value is required when vaultCert is enabled" .Values.vaultCert.key.path }}
+      key: {{ required "A valid vaultCert.key.secretKey value is required when vaultCert is enabled" .Values.vaultCert.key.secretKey }}
 {{ end -}}
```

Comment:
We are going to need to add keys for the intermediate certificate to this secret also. Then mount them to the apache proxy. They will likely need to be consumed in the same manner used by the legacy firecloud apache proxies

Reply:
The above is in order to use the old wildcard certificates from vault

Reply:
Done, thanks for pointing this out